src/math/Sphere.js

```diff
@@ -104,6 +104,20 @@ THREE.Sphere.prototype = {
 
 	},
 
+	intersectsPlane: function ( plane ) {
+
+		// We use the following equation to compute the signed distance from
+		// the center of the sphere to the plane.
+		//
+		// distance = q * n - d
+		//
+		// If this distance is greater than the radius of the sphere,
+		// then there is no intersection.
+
+		return ( ( this.center.dot( plane.normal ) - plane.constant ) <= this.radius );
+
```

Comment:
@WestLangley You're right, i've forgot to implement the absolute value of the distance! 

Oh, i didn't realize that there are unit tests for three.js! Sry, i will add some tests...